test/helpers.tsx

```diff
@@ -2,15 +2,18 @@ import React from "react";
 
 import { AuthProvider, AuthProviderProps } from "../src/AuthProvider";
 
-export const createWrapper = (opts: AuthProviderProps) => ({
-    children,
-}: React.PropsWithChildren<AuthProviderProps>): JSX.Element => (
-    <AuthProvider
-        {...opts}
-    >
-        {children}
-    </AuthProvider>
-);
+export const createWrapper =
+    (opts: AuthProviderProps, strictMode=false) =>
```

Comment:
I think that the default should be `true`.
In most cases, you would want to get the exact behavior both in `StrictMode` and production.

The only caveat here is in `useEffects`, if you want to check the number of times a function has been called,.

```suggestion
    (opts: AuthProviderProps, strictMode=true) =>
```